slemicro/create_vm.sh

```diff
@@ -101,8 +101,17 @@ if [ -f ${BASEDIR}/combustion/script ]; then
 		export SSHCONFIG=true
 		export SSHCONTENT=$(cat ${SSHPUB})
 	fi
-	# Copy it to the final iso destination parsing the vars
+
+	# Parse the file and copy it to the final ISO
 	envsubst < ${BASEDIR}/combustion/script > ${TMPDIR}/combustion/script
+	
+	# Copy all combustion related files to the final iso destination parsing the vars
```

Comment:
At the end of the day it doesn't hurt to have the scripts there if they are not going to be executed and it makes things easier (parse the vars and copying them)
Checking if every script needs to be added adds a big layer of complexity (you need to have a check for each one... How? A big switch? An If for each one? A naming convention?). The logic is already in the scripts and it will add more logic to the script just to save a few bytes..., but if you feel like doing it, I'm all ears 😃

Reply:
Also, when a new script needs to be added, the create VM needs to be edited as well... And also we did this is because we wanted that any user can drop any script there and it will be executed... So if they need to add logic to both places is pointless IMHO.

Reply:
Let me explain with examples:
- Scripts are scripts to exec something in a generic way. Don't including logic about .env make it generic as well as re-usable. You could add some pieces of code and maybe reusing from another scripts without adding any logic. Scripts should be scripts without any logic.
- The logic about the environment should be deployed is in create_vm.sh with the .env file, so the logic should be there in order to build the final combustion script just with the steps are required, not more. Imagine we've have 1000 snippets... why we need to add all of them there...
- Regarding the extra scripts you could have an 99_extra.sh where the user's snippets should be all there, and just making a flag in env to add extra_steps = true could be fixed.
- So, definitely I will add the logic here:
`
[[ ${PODMAN} = true ]] && echo "./09_podman.sh" >> ${TMPDIR}/combustion/script;
`
`
[[ ${ELEMENTAL} = true ]] && echo "./09_elemental.sh" >> ${TMPDIR}/combustion/script;  
...
`

Reply:
after talking internally we've got an agreetment